components/Footer/index.js

```diff
@@ -8,7 +8,9 @@ class Footer extends React.Component {
   render(): ?ReactElement {
     return (
       <footer className="Footer">
-        <p className="Footer-legal">Ⓒ 2015 Skookum. All Rights Reserved.</p>
+        <div className="InnerMax">
```

Comment:
Rather than mixing Suit components (in this case `Footer` and `InnerMax`) within the same component, we should either:

1. Create a secondary `InnerMax/LayoutConstraint` component that handles that concern, or
2. Handle the max width constraints independently per component per section.